crosslink/internal/common.go

```diff
@@ -53,3 +55,21 @@ func writeModule(module *moduleInfo) error {
 
 	return nil
 }
+
+func forGoModules(logger *zap.Logger, rootPath string, fn func(path string) error) error {
```

Comment:
Based on the name, I would expect this to iterate over values of the form `path/to/module` instead of `path/to/module/go.mod`, but maybe that's just me :)

Reply:
I will add a comment describing this 😉

Reply:
https://github.com/open-telemetry/opentelemetry-go-build-tools/pull/311/commits/fb5235c532b1649b8a21473c2465ab9f79bc4d50